### PR TITLE
New frontend

### PR DESCRIPTION
**Pull request by Neamar**
Related issues & design: #127
### HTML
- **Global**
  - [x] #132 Main layout & pods
- **Wallstreet**
  - [x] #120 Wallstreet
  - [x] #123 Corporation detail page
- **Players**
  - [x] #124 Players page
  - [x] #133 Player detail page
- **Orders**
  - [x] #129 Orders
- **Rules**
  - [x] #126 Rules
  - [x] #166 Corporation documentation
### CSS
- **Global**
  - [x] #132 Main layout & pods
- **Wallstreet**
  - [x] #120 Wallstreet
  - [x] #123 Corporation detail page
- **Players**
  - [x] #124 Players page
  - [x] #133 Player detail page
- **Orders**
  - [x] #129 Orders
- **Rules**
  - [x] #126 Rules
  - [x] #166 Corporation documentation

### CODE REVIEW

File: bin/install_nodejs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```
Comment (Amoki) [inline bin/install_nodejs]: you can't replace this file with multi buildpack ? It provides node and npm, even if there is no package.json

Comment (Neamar) [inline bin/install_nodejs]: ==> #153

**Comment by Neamar** [comment]
Can we merge this back into master?

**Comment by Amoki** [comment]
Says the guy that is bored by the flood

**Comment by Neamar** [comment]
Yup.

Plus, I want to see this in our commits stats!

**Pull request merged**